contracts/src/UniswapV3Sampler.sol

```diff
@@ -346,10 +256,10 @@ contract UniswapV3Sampler {
             }
         }
         // Must have a balance of both tokens.
-        if (pool.token0().balanceOf(address(pool)) == 0) {
+        if (IERC20TokenV06(pool.token0()).balanceOf(address(pool)) == 0) {
```

Comment:
just curious -- how come this changed?

Reply:
I removed the import for `IERC20TokenV06` in `IUniswapV3.sol`. This is the only line where we use `IERC20TokenV06` functions so didn't think it was needed as an import for the entire set of interfaces.